storage/tables/index.md

```diff
@@ -152,12 +152,53 @@ the result table will contain:
 |John|15|$150|
 |Darla|60|$600000|
 |John|34|$340000|
-|Annie|30|$500|
+|Annie|30|$500000|
 
-When importing data into a table with a primary key, the uniqueness is checked. 
+When importing data into a table with a primary key, the uniqueness is checked.
 The record `John,34,$340000` will overwrite the row `John,34,$340`, because it has the same primary key.
-The above applies only when **incremental load** is used. 
+The above applies only when **incremental load** is used.
+
+If an imported record does match an existing row but does not contain any change in values
+(eg. importing `John,34,$340`) the record will not be updated.
 
-When incremental is not used, the contents of the target table are cleared before the load. When a primary key 
+When incremental is not used, the contents of the target table are cleared before the load. When a primary key
 is not defined and an incremental load is used, it simply appends the data to the table and does not update anything.
 
+### Incremental Processing
+When a table is loaded incrementally the update time of each row is recorded internally. This information
+can be later used in Input Mapping of many components (especially [Transformations](/manipulation/transformations/mappings/#input-mapping)).
+In input mapping, you may specify the `Changed in last` option which allows the component to process only an increment of the data:
+
+{: .image-popup}
+![Screenshot - Incremental Processing](/storage/tables/incremental-processing.png)
+
+Extending the [above example](#incremental-loading) - if you have a table with a primary key defined on the columns `name` and `age`:
+
+|name|age|money|
+|---|---|---|
+|John|15|$150|

```

Comment:
reálnější hodnoty

Reply:
co jsou realnejsi hodnoty?

Reply:
no je tam jeden John se $150 a druhej s $340000, tak mi přijde zjevný, že to jsou dummy data a nemaj smysl (věrohodnost)

Reply:
vsak to maji byt dummy data